### PR TITLE
fix/rr415-reduce-size-of-qrcode

### PR DESCRIPTION
**Pull request by JacOng17**
# What has changed and why?
Changed size & set qr code to appear in the center of page.

# How to test this change
- [ ] at least one a reviewer ran the code

### CODE REVIEW

File: app/assets/stylesheets/admin/rebate_form.css [hunk 1]
```diff
@@ -227,8 +227,14 @@
 
 .rebate-qrcode {
   display: flex;
-  margin: 1em auto;
+  margin: 5em auto;
   background-color: black;
+  width: 20%;
```
Comment (mischa-s) [inline app/assets/stylesheets/admin/rebate_form.css]: Are these lines just for centering it on the page?
```
 position: absolute;
  top: 50%;
  left: 50%;
  margin-right: -50%;
  transform: translate(-50%, -50%);
```

Comment (JacOng17) [reply app/assets/stylesheets/admin/rebate_form.css]: Yup :stuck_out_tongue:

**Comment by JacOng17** [comment]
Have added a min & max width. @mischa-s

**Comment by mischa-s** [comment]
min and max widths should be absolute px, not relative
```
  width: 25%;
  min-width: 350px;
  max-width: 500px;
```

File: app/assets/stylesheets/admin/rebate_form.css [hunk 2]
```diff
@@ -227,8 +227,16 @@
...
+  min-width: 350px;
+  max-width: 500px;
```
Comment (Br3nda) [inline app/assets/stylesheets/admin/rebate_form.css]: :sparkles:

File: app/assets/stylesheets/admin/rebate_form.css [hunk 3]
```diff
@@ -229,8 +229,16 @@
```
Comment (mischa-s) [inline app/assets/stylesheets/admin/rebate_form.css]: suggest

```
.rebate-qrcode {
  margin: 5em auto;
  background-color: black;
  width: 20%;
  min-width: 350px;
  max-width: 500px;
  position: absolute;
  top: 50%;
  left: 50%;
  transform: translate(-50%, -50%);
}
```

Comment (mischa-s) [reply app/assets/stylesheets/admin/rebate_form.css]: :sparkles:  I've had some time to investigate this a bit further this morning. Now that I understand the constraints of this issue, I think the general approach you've gone with is the right one. However a good thing to do when you end up with CSS like this is to try removing each line in turn to make sure they are all active. In this case I found that neither `display: flex` or `margin-right` are having an impact so lets remove those and then git this merge!!! :sparkles:

**Pull request merged**